firstvoices/backend/serializers/fields.py

```diff
@@ -92,6 +92,21 @@ def to_representation(self, value):
             raise serializers.ValidationError("Invalid visibility value.")
 
 
+class WritableRoleField(serializers.CharField):
```

Comment:
I think you should be able to use EnumField (directly below) instead of creating a new field for this. That's what we use for Visibility, which is the same kind of data as Role.

Reply:
Actually, aren't we using the WritableVisibilityField for Visibility? (directly above)

Reply:
Oh dear, looks like we are using a mix of both. I was looking at this spot where we use EnumField: https://github.com/First-Peoples-Cultural-Council/fv-be/blob/87861203fd0fb8be538aaea5e386cdccdd0edcef/firstvoices/backend/serializers/job_serializers.py#L96 

If the generic EnumField works for this case, let's use it, and at some point work on moving the rest of the visibility fields to use it as well. I don't think the bespoke versions are doing anything different, but correct me if I'm missing something

Reply:
The EnumField does a couple of things differently. The first being that It doesn't allow for labels with spaces (such as "Language Admin") which is fairly easily accommodated without affecting other uses of the field by adding `.replace(" ", "_")`. The second difference is it strips casing from the labels it returns (`.lower()`) and many other tests for other apis are expecting it to do this. Membership is currently cased both in the fe and in `get_role_display`.